easybuild/easyconfigs/r/Ruby/Ruby-2.7.1-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,158 @@
+name = 'Ruby'
+version = '2.7.1'
```

Comment:
@jedokaplan We should go with the latest Ruby version, done in https://github.com/jedokaplan/easybuild-easyconfigs/pull/1

Reply:
@jedokaplan ping change request above?

Reply:
fixed in e75752f